src/core/utils.js

```diff
@@ -97,6 +97,10 @@ const fetchDestinations = new Set([
   "",
 ]);
 
+//these classes mean a section is non-normative
+export const nonNormativeClasses =
```

Comment:
This being a CSS selector (and "classes" being ambiguous), I'd suggest: 

```suggestion
export const nonNormativeSelector =
```